docs/example.ts

```diff
@@ -27,10 +42,20 @@ async function setup(): Promise<{
 
   // To get an attestation, we need an Attester
   // we can generate a new keypair:
-  // const attester = await Kilt.AttesterIdentity.buildFromMnemonic("...")
+
+  const keyring = new Kilt.Utils.Keyring({
+    // KILT has registered the ss58 prefix 38
+    ss58Format: 38,
+    type: 'ed25519',
+  })
+  // generate a Mnemonic for the attester
+  // const generateAttesterMnemonic = Kilt.Utils.UUID.generate()
+  const attesterMnemonic =
+    'receive clutch item involve chaos clutch furnace arrest claw isolate okay together'
+
   // or we just use unsafe precalculated keys (just for demo purposes!):
-  const attester = Kilt.Identity.buildFromMnemonic(
-    'receive clutch item involve chaos clutch furnace arrest claw isolate okay together',
+  const attester = keyring.addFromMnemonic(
+    attesterMnemonic,
     // using ed25519 bc this identity has test coins from the start on the development chain spec
     { signingKeyPairType: 'ed25519' }
```

Comment:
Same here?

Reply:
I wanted to keep it consistent throughout.